plugin/storage/kafka/options.go

```diff
@@ -33,14 +35,21 @@ const (
 	// EncodingZipkinThrift is used for spans encoded as Zipkin Thrift.
 	EncodingZipkinThrift = "zipkin-thrift"
 
-	configPrefix          = "kafka.producer"
-	suffixBrokers         = ".brokers"
-	suffixTopic           = ".topic"
-	suffixProtocolVersion = ".protocol-version"
-	suffixEncoding        = ".encoding"
-	defaultBroker         = "127.0.0.1:9092"
-	defaultTopic          = "jaeger-spans"
-	defaultEncoding       = EncodingProto
+	configPrefix           = "kafka.producer"
+	suffixBrokers          = ".brokers"
+	suffixTopic            = ".topic"
+	suffixEncoding         = ".encoding"
+	suffixRequiredAcks     = ".required.acks"
```

Comment:
this should be `required-acks` (with a dash, not a dot, because it's not a hierarchical struct, it's a single term)

Reply:
done